content/05.dl-complexities.md

```diff
@@ -1 +1,14 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with different tweaks and modifications.
+Despite improved framework ease-of-use and on-demand cloud computing resources this means DL can be very demanding, requiring significant infrastructure and patience to complete.
```

Comment:
I think we need to add a comparison here, because that's of course very relative. Just attended a seminar last week where someone ran simulations on 4096 CPUs that took months to complete ... the throughput was 1 petaflop, which could have been easily achieved via a handful of GPUs -- what I am trying to say is that the infrastructure is relatively demanding compared to maybe traditional machine learning on small datasets. Also, traditional MCMC or all the molecular dynamics research is much more expensive than DL (e.g., I would say DL is currently probably the cheapest method for protein structure refinement compared to traditional methods :P)

Reply:
I couldn't really find a good general comparison but I did find a paper that was looking at scaling and computational performance.  We don't need to say necessarily it is WORSE than other methods to have a strong case that it can still require significant infrastructure etc.